hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/server/SCMStorageConfig.java

```diff
@@ -77,4 +78,20 @@ protected Properties getNodeProperties() {
     return scmProperties;
   }
 
+  /**
+   * Sets the SCM CA certificate serial id.
```

Comment:
can you highlight this is for sub scm ca cert id.

Reply:
Done